mythril/laser/ethereum/state/calldata.py

```diff
@@ -21,84 +11,133 @@ class CalldataType(Enum):
     SYMBOLIC = 2
```

Comment:
Can we remove `CalldataType`? I believe the only place it is used is line 1270 of instructions.py, and that whole if statement is always false and should be deleted...

Reply:
See #556 ^^